portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -60,17 +65,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String text = getParameter(request, "user-comment", "");
     long timestamp = System.currentTimeMillis();
 
-    Entity taskEntity = new Entity("Comment");
-    taskEntity.setProperty("text", text);
-    taskEntity.setProperty("timestamp", timestamp);
+    Entity commentEntity = new Entity("Comment");
+    commentEntity.setProperty("text", text);
+    commentEntity.setProperty("timestamp", timestamp);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    datastore.put(taskEntity);
+    datastore.put(commentEntity);
 
     // Redirect back to the HTML page.
     response.sendRedirect("/index.html");
   }
 
+  /** Returns the number of comments the user has chosen to display */
+  private int getAmount(HttpServletRequest request) {
```

Comment:
Maybe name the function `getNumberOfComments` or `getNumberOfCommentsRequested` to be more clear it is the number of comments rather than something else?

Reply:
Will do!